src/main/docker/Dockerfile

```diff
@@ -14,40 +14,52 @@
 # limitations under the License.
 #
 
+# Preliminary image that includes Elasticsearch and related files
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.3.2 AS builder
+
 #
-# Preliminary image that downloads and extracts Elasticsearch
+# The actual image definition
 #
-FROM cafapi/opensuse-jre8:1 AS builder
+FROM cafapi/opensuse-jre8:1
 
-RUN zypper -n install which hostname
-RUN zypper -n install unzip
+ENV ELASTIC_CONTAINER=true
 
-# Download and Extract Elasticsearch
-ADD https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.3.2.zip /opt/
+COPY --from=builder /opt/jdk-10.0.2/ /opt/jdk-10.0.2/
+ENV JAVA_HOME=/opt/jdk-10.0.2
 
-RUN unzip /opt/elasticsearch-6.3.2.zip -d /opt/
+# Copy java certs and set link to them
+COPY --from=builder /etc/pki/ca-trust /etc/pki/ca-trust
+RUN ln -sf /etc/pki/ca-trust/extracted/java/cacerts /opt/jdk-10.0.2/lib/security/cacerts
 
-# Install the analysis-icu plugin for Elasticsearch
-RUN /opt/elasticsearch-6.3.2/bin/elasticsearch-plugin install analysis-icu
+# Instal packages
+RUN zypper --non-interactive -n install which hostname netcat-openbsd unzip wget
```

Comment:
`--non-interactive` and `-n` mean the same thing.  Just use `-n` since that's what we use in the other images.

Reply:
**-n** means to search packages by their name.

For **--non-interactive** there is also  alias **-y**, but the one I used tells more about what is it doing.

Reply:
Not according to `zypper --help`:

    92c7c68927e3:/ # zypper --help
      Usage:
            zypper [--global-options] <command> [--command-options] [arguments]
            zypper <subcommand> [--command-options] [arguments]

      Global Options:
            --help, -h              Help.
            --version, -V           Output the version number.
            --promptids             Output a list of zypper's user prompts.
            --config, -c <file>     Use specified config file instead of the default.
            --userdata <string>     User defined transaction id used in history and plugins.
            --quiet, -q             Suppress normal output, print only error
                                    messages.
            --verbose, -v           Increase verbosity.
            --color
            --no-color              Whether to use colors in output if tty supports it.
            --no-abbrev, -A         Do not abbreviate text in tables.
            --table-style, -s       Table style (integer).
            --non-interactive, -n   Do not ask anything, use default answers
                                    automatically.
            --non-interactive-include-reboot-patches
                                    Do not treat patches as interactive, which have
                                    the rebootSuggested-flag set.
            --xmlout, -x            Switch to XML output.
            --ignore-unknown, -i    Ignore unknown packages.

            --reposd-dir, -D <dir>  Use alternative repository definition file
                                    directory.
            --cache-dir, -C <dir>   Use alternative directory for all caches.
            --raw-cache-dir <dir>   Use alternative raw meta-data cache directory.
            --solv-cache-dir <dir>  Use alternative solv file cache directory.
            --pkg-cache-dir <dir>   Use alternative package cache directory.

Reply:
I used https://en.opensuse.org/SDB:Zypper_manual_(plain) to look for (different) options.